packages/core/src/codewhisperer/models/model.ts

```diff
@@ -303,7 +303,9 @@ export class TransformByQState {
 
     private payloadFilePath: string = ''
 
-    private jobFailureErrorMessage: string = ''
+    private jobFailureErrorNotification: string | undefined = undefined
+
+    private jobFailureErrorChatMessage: string | undefined = undefined
 
```

Comment:
Need separate messages for notifications and for chat messages since they differ slightly